### DISCUSSION TITLE
Anyway to download images from a Flair?

### DISCUSSION THREAD

**Discussion by azur1111** [Serene-Arc/bulk-downloader-for-reddit#315] [Q&A] [answered]
Basically on subreddits they have flairs with certain things for certain stuff. Anyway to download images within a flair?

**Comment by Serene-Arc** [comment, marked as answer]
Yes, you can use all of the fields outlined [here](https://www.reddit.com/wiki/search) in the BDFR. For example, `--search 'flair:example flair'` will find all submissions with the flair "example flair". Hope this helps :)
